backend/hasura-actions/src/Scheduler.res

```diff
@@ -5,31 +5,33 @@ type makePaymentRequest = {
   amount: string,
   identifier: option<string>,
 }
-
-// let dummyData = [
-//   {
-//     recipient: "0xc788F08a2aAf539111e2a2D85BD4B324FBE37B15",
-//     addressTokenStream: "0xb38981469B7235c42DDa836295bE8825Eb4A6389",
-//     lengthOfPayment: 86400, // seconds [86400 equals one day.] Must be a multiple of 60
-//     interval: 60, // this will always be 60 for our demo
-//     // TODO: these values should be BigInt and use `@decco.codec` as the conversion function
-//     rate: "1",
-//     deposit: "7200",
-//     numerOfPaymentsMade: 0,
-//     totalNumberOfPaymentsToMake: 1440,
-//   },
-//   // {
-//   //   recipient: "0xc788F08a2aAf539111e2a2D85BD4B324FBE37B15",
-//   //   addressTokenStream: "0xb38981469B7235c42DDa836295bE8825Eb4A6389",
-//   //   lengthOfPayment: 86400, // seconds [86400 equals one day.] Must be a multiple of 60
-//   //   interval: 60, // this will always be 60 for our demo
-//   //   // TODO: these values should be BigInt and use `@decco.codec` as the conversion function
-//   //   rate: "1",
-//   //   deposit: "14400",
-//   //   numerOfPaymentsMade: 0,
-//   //   totalNumberOfPaymentsToMake: 1440,
-//   // },
-// |];
+/*
+KEPT THIS AS A REMINDER FOR THE DEPOSIT
+let dummyData = [
+  {
+    recipient: "0xc788F08a2aAf539111e2a2D85BD4B324FBE37B15",
+    addressTokenStream: "0xb38981469B7235c42DDa836295bE8825Eb4A6389",
+    lengthOfPayment: 86400, // seconds [86400 equals one day.] Must be a multiple of 60
+    interval: 60, // this will always be 60 for our demo
+    // TODO: these values should be BigInt and use `@decco.codec` as the conversion function
+    rate: "1",
+    deposit: "7200",
+    numerOfPaymentsMade: 0,
+    totalNumberOfPaymentsToMake: 1440,
+  },
+  {
+    recipient: "0xc788F08a2aAf539111e2a2D85BD4B324FBE37B15",
+    addressTokenStream: "0xb38981469B7235c42DDa836295bE8825Eb4A6389",
+    lengthOfPayment: 86400, // seconds [86400 equals one day.] Must be a multiple of 60
+    interval: 60, // this will always be 60 for our demo
+    // TODO: these values should be BigInt and use `@decco.codec` as the conversion function
+    rate: "1",
+    deposit: "14400",
+    numerOfPaymentsMade: 0,
+    totalNumberOfPaymentsToMake: 1440,
+  },
+|];
```

Comment:
I guess you are going to take these for the process of creating a channel/depositing into it initially?

Reply:
Yeah they were the dummy data from the your prototype version, thought it would be useful to keep.